### PR TITLE
Milestone 3

### PR DESCRIPTION
**Pull request by Naqhid**
Here it includes Game logic

### CODE REVIEW

**Review by krys2fa — CHANGES_REQUESTED** [review]
## Great work @jamestucker1988  and @Naqhid  👏 
Your project is coming along nicely. 👍 
Just a few more adjustments to complete the milestone.
### Status: Changes Required ♻️ 

- [REQUIRED] Your pull request does not have GitHub Actions enabled. Please fix this.

- [REQUIRED] No `puts/print` and `gets.chomp` outside `main.rb` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/)).

- [ ] [REQUIRED] Fix all linter errors locally.

- [ ] [REQUIRED] The game does not run but gives the following error:
```
bin/main.rb:18: syntax error, unexpected `end', expecting end-of-input
    end
```
Please request for another review after implementing the changes.

Happy Coding 🖥️  😃 

###### You can contact me on slack (Chris Amanor) for any clarification.

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi Team     
#### Great Job so far, everything works fine, but there are a few more things required. 
### General
- [ ] I get this error whenever I try to run the app 
````ruby
Ruby-TicTacToe-milestone-3/lib/game.rb:21:in `choose_signs': undefined local variable or method `show_progress' for Game:Class (NameError)
````
- [ ] Let's activate linter checks for this project
### Make changes and submit another code review request.
*Happy coding* 
(Any questions Feel free to contact me on slack @kobiyoyo)

File: lib/board.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
+class Board
+  attr_reader :squares, :wining_combinations
+  attr_writer :squares
+
```
Comment (bertil291utn) [inline lib/board.rb]: ## attr_accesor
Due to you're using `attr_reader` and `attr_writer` accessors use directly `attr_accessor `

**Review by bertil291utn — CHANGES_REQUESTED** [review]
# **Status:** Changes requested 🟠 
<p align="center"> image <br><strong>Not yet</strong> </p>

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

### Your work is amazing you've mastered it GitFlow Flow you have a professional README file. 

However, in order to accomplish as the actual page, you required to do these changes, listed below:

### General requirements 
- Fix linters errors

### Summary
- Please linter errors
-  Don't forget to work with inline comments below this comment box.

***

Guys work in the requested changes above and request another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

**Review by Temmarie — CHANGES_REQUESTED** [review]
### Changes Required! :recycle:
Hi Team! 👋 👋 

### You have done well so far :clap: However, you did not finish all the previous review changes. Kindly work on them and submit them for another code review.

**Recommended**

- [ ] Please fix all linter errors. The current linter error you have is as a result of you not making your `main.rb` file executable.

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink:
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

**Review by ebeagusamuel — APPROVED** [review]
# Project Approved :100: 
Hello team :wave: 
Congratulations on meeting the requirements for this project and getting it approved. :dancing_men: :dancing_women: 
You can now merge this branch

slack(@ebeagusamuel)

**Pull request merged**